lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -2,6 +2,14 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmCloud
     expose :cloud_networks, :association => true
 
+    expose :resize
```

Comment:
I think I was doing this before and I was able to call the methods from automate without problems. Though you will be sending there Flavor Automate Model, so make sure it acts the same as Flavor in that method. :-)